internal/provider/data_env_variables_test.go

```diff
@@ -8,15 +8,19 @@ import (
 )
 
 func TestDataVariables(t *testing.T) {
+	arn := "arn:aws:secretsmanager:us-east-1:522657839841:secret:scarlet-eagle-kvoty/conn_url-lPd8oL"
```

Comment:
Let's use fake ARNs and fake account IDs.

Reply:
Done